src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,94 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and publish your app, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When creators edit their Koji profile, they can choose to add an app from the App Store.
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
+
+== Preparing your marketing assets
+
+You need the following marketing assets for your App Store listing:
+
+* A 500x500-pixel icon to be displayed with your listing in the App Store.
+* A 1200x630-pixel thumbnail to be displayed when your listing is shared in other social media sites.
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+TIP: For a smoother process, prepare your marketing assets before you create your listing.
+
+== Creating an App Store listing
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal]
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
+
+. Click *New App Template*.
+
+. Give your listing a name.
++
+This name is not published in the App Store; it is used to identify your listing.
+
+. Fill in the *App Template Details*.
++
+IMPORTANT: All texts and images in your listing must meet https://withkoji.com/resources/terms[Koji's Terms of Use] requirements.
++
+* *Icon* – The 500x500-pixel icon displayed with your listing.
+
+* *Name* – The title of your listing.
+Note that the title could be truncated depending on the width of the browser page; therefore, place distinguishing words near the beginning.
+
```

Comment:
This is missing "Subtitle," which is the small tagline that appears underneath the app name when the app is viewed as part of a category:
![image](https://user-images.githubusercontent.com/3889063/122424178-5cb42380-cf43-11eb-881e-205a91341df4.png)

Reply:
Added:
> * *Subtitle* – A short tagline that appears below the app name in category lists.